library/alloc/src/vec/peek_mut.rs

```diff
@@ -11,19 +12,19 @@ use crate::fmt;
 ///
 /// [`peek_mut`]: Vec::peek_mut
 #[unstable(feature = "vec_peek_mut", issue = "122742")]
-pub struct PeekMut<'a, T> {
-    vec: &'a mut Vec<T>,
+pub struct PeekMut<'a, T, A: Allocator> {
```

Comment:
The allocator parameter should be unstable and default to `Global`, just like that of `Vec`,